### PR TITLE
Register an array of actions

### PR DESCRIPTION
**Pull request by mikeykhalil**
I believe we spoke offline about implementing this feature. Let me know if this is not what you had in mind.

### CODE REVIEW

**Comment by robgonnella** [comment]
One small comment otherwise this looks great to me!

**Comment by mikeykhalil** [comment]
Sorry, not sure if I see the comment. Though it's possible it's just because I am not used to Github's interface 👶

File: src/index.ts [hunk 1]
```diff
@@ -84,7 +93,7 @@ export class ReduxAnalyticsManager<A, S> {
 
     this.initialized = true;
 
-    return (store) => (next) => (action) => {
+    return (store) => (next: Dispatch<AnyAction>) => (action: AnyAction) => {
```
Comment (robgonnella) [inline src/index.ts]: Thanks for adding these types! I think we can add `MiddlewareAPI` as the type for store: https://github.com/reduxjs/redux/blob/master/index.d.ts#L303

**Comment by robgonnella** [comment]
That was my fault, I clicked start a review instead of leave comment. Sorry about that.

**Comment by mikeykhalil** [comment]
Ah, yeah, I see what you mean. That is a weird feature. I think I prefer Gitlab's workflow for code reviews a bit more. 

And good catch about the additional type information! I got frustrated trying to add the type for `store` earlier, but I am just now realizing that I was trying to assign the wrong type to that argument.

**Comment by robgonnella** [comment]
LGTM!

**Pull request merged**